README.md

```diff
@@ -1,23 +1,75 @@
-Python courses for the scientific researcher
-============================================
+# Python courses for the scientific researcher
 
 This repository contains several courses for the benefit of scientific researchers,
```

Comment:
I know this isn't your sentence, but I think the word 'software' in between 'several' and 'courses' would be nice.